app/controllers/keywords_controller.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+class KeywordsController < ApplicationController
+  before_action :authenticate_user!
+
+  def index
+    @keywords = Keyword.all
+    @keyword = Keyword.new
```

Comment:
Let's use local variable here

```suggestion
    keywords = Keyword.all
    keyword = Keyword.new

    render: :index, locals: { keywords: keywords, keyword: keyword } 
```

Reply:
Yup! I went with the `Presenter` [convention](https://nimblehq.co/compass/development/code-conventions/ruby/ruby-on-rails/#presenters) as suggested earlier 😄